python/stlib/components/__init__.py

```diff
@@ -13,32 +13,37 @@
 
 
 """
-__all__=["all"]
+__all__ = ["all"]
 
 from splib.numerics import getOrientedBoxFromTransform
 
-def addOrientedBoxRoi(parentNode, position, name="BoxRoi", translation=[0.0,0.0,0.0], eulerRotation=[0.0,0.0,0.0], scale=[1.0,1.0,1.0]):
-    orientedBox = getOrientedBoxFromTransform(translation,eulerRotation=eulerRotation,scale=scale)
-    return parentNode.createObject("BoxROI", position=position, orientedBox=orientedBox, drawBoxes=True )
 
-def OrientedBoxFromTransform(translation=[0.0,0.0,0.0], eulerRotation=[0.0,0.0,0.0], scale=[1.0,1.0,1.0]):
-    raise Exception("This function is now deprecated and replaced with splib.numerics.getOrientedBoxFromTransform Please update your code.")
+def addOrientedBoxRoi(parentNode, position, name="BoxRoi", translation=[0., 0., 0.], eulerRotation=[0., 0., 0.], scale=[1., 1., 1.], drawBoxes=True):
+    orientedBox = getOrientedBoxFromTransform(translation=translation, eulerRotation=eulerRotation, scale=scale)
+    return parentNode.createObject("BoxROI", position=position, orientedBox=orientedBox, drawBoxes=drawBoxes)
```

Comment:
I also added an option to draw the boxes or not. I didn't change the default value.